### PR TITLE
added solution to the hw

### PR DESCRIPTION
**Pull request by AnyaRatusnyak**

### CODE REVIEW

File: src/main/java/TicketService.java [hunk 1]
```diff
@@ -122,4 +132,24 @@ private static Ticket[] findByStadiumSector(String sector, Ticket[] tickets) {
         }
         return ticketsInSector.toArray(new Ticket[0]);
     }
+
+    private static ArrayList<Map<String, Object>> loadTicketsToArrayList(ApplicationContext context, String path) {
```
Comment (Alexmansar) [inline src/main/java/TicketService.java]: List<Map<String, Object>>, not ArrayList<Map<String, Object>>

File: src/main/java/TicketService.java [hunk 2]
```diff
@@ -122,4 +132,24 @@ private static Ticket[] findByStadiumSector(String sector, Ticket[] tickets) {
...
+                ArrayList<Map<String, Object>> list = gson.fromJson(reader, listType);
+                return list;
+
+            } catch (IOException e) {
+                throw new RuntimeException("Error reading the resource", e);
+            }
+        } else {
+            throw new RuntimeException("File not found: " + path);
+        }
+    }
```
Comment (Alexmansar) [inline src/main/java/TicketService.java]: please, use it in the separate class.

File: src/main/java/dao/UserDataBaseDaoImpl.java [hunk 3]
```diff
@@ -98,4 +104,33 @@ public boolean delete(Long id) {
...
     }
+
+    @Transactional
+    @Override
+    public UserDataBase activateUser(UserDataBase userDataBase) {
+        if (!isUserActivationEnabled) {
+            throw new RuntimeException("User activation feature is turned OFF.");
+        }
+        try (Session session = sessionFactory.openSession()) {
+            Transaction transaction = session.beginTransaction();
```
Comment (Alexmansar) [inline src/main/java/dao/UserDataBaseDaoImpl.java]: do we need it? we use annotation here. Please, check it)

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@AnyaRatusnyak Hi) in general good solution, just few comments)

File: src/main/java/TicketService.java (hunk 1 above)
Comment (AnyaRatusnyak) [reply src/main/java/TicketService.java]: Good morning ) Sorry

File: src/main/java/TicketService.java (hunk 2 above)
Comment (AnyaRatusnyak) [reply src/main/java/TicketService.java]: Ok

File: src/main/java/dao/UserDataBaseDaoImpl.java (hunk 3 above)
Comment (AnyaRatusnyak) [reply src/main/java/dao/UserDataBaseDaoImpl.java]: Oh! My mistake (( Annotation @Transactional include session.beginTransaction() and commit() and rollback().
Thank you ! Good day

**Review by Alexmansar — APPROVED** [review]
@AnyaRatusnyak Hi)) Good job) could be merged) have a nice day too)

**Pull request merged**